Gemfile.lock

```diff
@@ -343,8 +345,8 @@ GEM
       rack-protection (~> 1.4)
       tilt (>= 1.3, < 3)
     slop (3.6.0)
-    sprockets (3.3.4)
```

Comment:
Manual merge error. A sprockets update isn't relevant to the purpose of this PR, but it doesn't hurt.